tests/test_llama_e2e.py

```diff
@@ -278,7 +278,7 @@ def test_llama_e2e_two_addtional_tokens(self):
     slot = 0
 
     # pylint: disable-next=all
-    prefill_result = engine.prefill(
+    prefill_result, _ = engine.prefill(
```

Comment:
Please add the `first_token` to the out_tokens array (Line 288), otherwise `out_tokens` will not equal `expected_output_tokens`. Same for all other calls..

Reply:
+1